examples/react-app/src/amplitude/index.ts

```diff
@@ -1,21 +1,16 @@
 import { AmplitudeLoadOptions as AmplitudeLoadOptionsCore, Logger, NoLogger } from "@amplitude/amplitude-core";
```

Comment:
Maybe we can add a commented notice showing that the file is auto-generated and shouldn't be modified manually for the generated files? I am not sure whether this file and `examples/feature-overview/amplitude/node.ts` and `examples/feature-overview/amplitude/browser.ts` are auto-generated or not 🤣.

Reply:
Thanks @liuyang1520  I will add the header. Was mostly focused on getting the codegen functional for Alpha, but agree we should get the DNM warning in sooner than later.

Reply:
Added DNM header.